pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcProduction.scala

```diff
@@ -35,7 +35,16 @@ object MarcProduction
       case (Nil, Nil)     => Nil
       case (from260, Nil) => from260
       case (Nil, from264) => from264
-      case (_, _)         => getProductionFromBothFields(record)
+      // If both 260 and 264 are present we prefer the 260 fields, see if we can
```

Comment:
This comment looks a bit truncated - perhaps this might be clearer.

```suggestion
      // If both 260 and 264 are present we prefer the 260 fields, see if we can safely ignore the 264 content
```